### ISSUE TITLE
How about Using this in Mobile ?

### ISSUE THREAD

**Issue by itskrsna** [simplymovein/angular2-useful-swiper#1]
On Orientation Change how to `reInit()` the plugin manually ?

**Comment by ahmedshabib** [comment]
Check the Swiper API,
There is an option called breakpoints, which allows you to configure different parameters for different screen sizes, it should solve your problem.

**Comment by itskrsna** [comment]
I don't think that is a convenient way to do because when we are targeting android then there will be having a large number of devices ..
